packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -324,6 +324,20 @@ mixin WidgetsBinding on BindingBase, ServicesBinding, SchedulerBinding, GestureB
         },
       );
 
+      // Register the ability to quickly mark elements as dirty.
+      registerServiceExtension(
+        name: 'fastReassemble',
+        callback: (Map<String, Object> params) async {
+          final String className = params['class'];
+          for (Element element in debugTrackedElements) {
```

Comment:
With local testing this was surprisingly fast

Reply:
So this is iterating through all the elements finding the ones that match a specific class name.
In the case of updating a Text or Container widget, wouldn't that trigger quite a lot of widgets to be updated?
Is the performance be about the same if instead of using
debugTrackedElements you traverse the element tree instead only calling `markNeedsBuild` when widgets match the class name?

Reply:
That sounds like a better idea actually, let me get some numbers

Reply:
Updated, noticed no significant difference, but traversing normally requires no additional APIs